#!/usr/bin/env python
PACKAGE = "dynamic_reconfigure_test"
 
# 导入dynamic_reconfigure功能包提供的参数生成器（parameter generator）
from dynamic_reconfigure.parameter_generator_catkin import *
 
# 创建一个参数生成器
gen = ParameterGenerator()
 
# 定义动态配置的参数
 
gen.add("K_s", double_t, 0, "K_s_param", 0.0,  0, 500)
gen.add("beita", double_t, 0, "beita_param", 0.0,  0, 500)
gen.add("alphe1", double_t, 0, "alphe1_param", 0.0,  0, 500)
gen.add("alphe2", double_t, 0, "alphe2_param", 0.0,  0, 500)

 
# 退出
exit(gen.generate(PACKAGE, "dynamic_RISE", "rise"))


